routetable/route_table.go

```diff
@@ -65,7 +65,8 @@ func init() {
 type TargetType string
 
 const (
-	TargetTypeVXLAN TargetType = "vxlan"
+	TargetTypeVXLAN       TargetType = "vxlan"
+	TargetTypeVXLANParent TargetType = "default"
```

Comment:
As I suggested earlier I think we should align the names here better so the variable name matches the value. It will be easier to follow that way. Probably remove VXLAN and Parent and go with something generic.

e.g. `TargetTypeDefault = "default"`